### ISSUE TITLE
Dimension $\xi$ breaks HoloMap

### ISSUE THREAD

**Issue by tjol** [ioam/holoviews#573]
Calling a dimension of a HoloMap or DynamicMap `$\xi$` fails with a javascript error
  `SyntaxError: malformed hexadecimal character escape sequence`

As far as I can tell, this is a question of insufficient escaping. Holoviews produces code like
``` javascript
    widget_ids[0] = "_anim_widget0f74e25011d543c28d2ea4d20346da74_$\xi$";`
```

which obviously isn't good JS.

**Comment by philippjfr** [comment]
Thanks @tjol, the dimension names should be getting escaped, not sure why that isn't happening. Is this using current master?

**Comment by jlstevens** [comment]
I think this is related to PR #476 for processing escape codes. Looks like ``\x`` should be in the list...

@tjol I think the solution is to use a raw string ``r'\xi'`` as ``\x`` is always going to cause trouble in regular strings...

**Comment by philippjfr** [comment]
We were already escaping dimensions passed to Javascript in some cases but it seems I overlooked this when implementing that. I've added the fix to #575 and closed #574.

**Comment by tjol** [comment]
@jlstevens this is a javascript error, not a python one. Of course I'm already using a raw string.

@philippjfr yes, this is with the current master. It also happens with version 1.4.3 from PyPI.

**Comment by philippjfr** [comment]
This has been fixed in #575 so I'll close. Please reopen if you encounter any issues.

**Comment by tjol** [comment]
Thanks for fixing!
